models/DataObject/SelectOptions/Config/Listing.php

```diff
@@ -0,0 +1,89 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Model\DataObject\SelectOptions\Config;
+
+use Pimcore\Model;
+
+/**
+ * @method \Pimcore\Model\DataObject\SelectOptions\Config\Listing\Dao getDao()
+ */
+class Listing extends Model\AbstractModel implements \IteratorAggregate
+{
+    /**
+     * @internal
```

Comment:
why internal?

Reply:
I think I copied it from the thumbnail setup.

Reply:
I'm not sure why it is like this in Thumbnail listing. Let's keep it then for consistency.